### PR TITLE
fix: repo URLs

### PR DESCRIPTION
**Pull request by alanshaw**
Fixes the repo URLs

### CODE REVIEW

File: src/pages/concepts/ucan.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ async function delegationRequestHandler (request) {
   // "*", which includes all abilities that your agent has access to.
   //
   // See the capabilities spec for more about capabilities:
-  // https://github.com/web3-storage/w3protocol/blob/main/spec/capabilities.md
+  // https://github.com/storacha/w3up/blob/main/spec/capabilities.md
```
Comment (fforbeck) [inline src/pages/concepts/ucan.md:115]: Got a 404 for this one.

File: src/pages/w3cli.mdx [hunk 2]
```diff
@@ -35,9 +35,9 @@ Once the install is complete, you'll have a `w3` command available. Try running
   These binaries are built from the Go client codebase which is not as fully featured as the JS client.
 </Callout>
 
-Pre-compiled, standalone binaries are now available for MacOS, Linux and Windows: [Download latest release](https://github.com/web3-storage/go-w3up/releases/latest).
+Pre-compiled, standalone binaries are now available for MacOS, Linux and Windows: [Download latest release](https://github.com/storacha/go-w3up/releases/latest).
 
-Please ensure you [read the documentation for using the Go CLI](https://github.com/web3-storage/go-w3up#cli). These binaries are built from the Go client codebase which is not as fully featured as the JS client. Currently to use the Go CLI tool you will need to obtain a delegation created by the JS CLI or [console](https://console..network).
+Please ensure you [read the documentation for using the Go CLI](https://github.com/storacha/go-w3up#cli). These binaries are built from the Go client codebase which is not as fully featured as the JS client. Currently to use the Go CLI tool you will need to obtain a delegation created by the JS CLI or [console](https://console..network).
```
Comment (fforbeck) [inline src/pages/w3cli.mdx:40]:
```suggestion
Please ensure you [read the documentation for using the Go CLI](https://github.com/storacha/go-w3up#cli). These binaries are built from the Go client codebase which is not as fully featured as the JS client. Currently to use the Go CLI tool you will need to obtain a delegation created by the JS CLI or [console](https://console.storacha.network).
```

**Review by fforbeck — CHANGES_REQUESTED** [review]
Overall it looks great. I just found two 404 errors which I pointed out in the comments.

File: src/pages/concepts/ucan.md (hunk 1 above)
Comment (alanshaw) [reply src/pages/concepts/ucan.md:115]: Oh great catch! Thank you!

**Review by fforbeck — APPROVED** [review]
LGTM!

**Pull request merged**